source/simulator/stokes_matrix_free.cc

```diff
@@ -1908,6 +1911,8 @@ namespace aspect
         const double imbalance = MGTools::workload_imbalance(sim.triangulation);
         sim.pcout << "    GMG workload imbalance: " << imbalance << std::endl
                   << "    Stokes solver: " << std::flush;
+
+        sim.pcout << "Viscosity range: " << viscosity_min << " - " << viscosity_max << std::endl;
```

Comment:
Move this new code above the line 1913. Right now you are printing in the middle of the line of the Stokes iterations.